src/Mailgun/Mailgun.php

```diff
@@ -36,20 +36,83 @@ class Mailgun
     protected $apiKey;
 
     /**
+     * @var [string => string]
+     */
+    protected $regions = [
+        'si' => 'si.api.mailgun.net',
+        'so' => 'so.api.mailgun.net',
+    ];
+
+    /**
+     * @var [string => RestClient]
```

Comment:
`@var array [string => RestClient]`

Right? Or else PHPDoc will not pick this up. Same with $regions.